### ISSUE TITLE
Account ID

### ISSUE THREAD

**Issue by thomas70** [bisho/carnet#1]
I cant find my Account ID. I login with email address, password and pin. Where can i find the ID ?

**Comment by thomas70** [comment]
seems like the api is for us and not europe ?

**Comment by bisho** [comment]
Yeah, the api might be US only. Let me know if you find how the EU api works, would be nice to add support for that too.

**Comment by Konstigt** [comment]
How did you get the information on how to use the US API? 

I have only email and password (Europe). I remember that I had a PIN before but that seems to be gone now. Maybe because I haven't got an Android Wear device any longer or maybe just the app has been updated. Also checked www.volkswagen.com/car-net but I could not find ID nor PIN there.

Tried capturing a login from the app but as expected I could only see HTTPS-traffic. The app connects, https://play.google.com/store/apps/details?id=de.volkswagen.carnet.eu.eremote,  to https://143.164.100.37/

**Comment by borisgodet** [comment]
Hi,
Same here, tried with a FR account but does not connect with the credentials (email + pwd)
Anybody has any clue regarding EU connecting ?
Thanks

**Comment by jaap** [comment]
Same here. It would be nice to have this working in Europe as well. I would be happy to help out/test on my car.
